app/_components/commodities/Form.tsx

```diff
@@ -136,12 +145,22 @@ const Form: React.FC<FormProps> = ({ onSubmitHandler, isLoading }) => {
               ))}
             </Stack>
           </CheckboxGroup>
-          <FormLabel>Max Landing Pad Size</FormLabel>
+          <FormLabel>Landing Pad Size</FormLabel>
           <RadioGroup>
-            <Stack spacing={8} direction="row" mt={8} ml={8} flexWrap="wrap">
+            <Stack
+              borderWidth="2px"
+              borderRadius="9px"
+              borderColor={selectColor(isDark, 'text')}
+              p="1rem"
+              spacing={8}
+              direction="row"
```

Comment:
```suggestion
              direction={['column', 'row']}
```
This and the proceeding `<Stack>` element should be similar across viewports.